src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala

```diff
@@ -0,0 +1,40 @@
+package uk.gov.nationalarchives.api.update
+
+import com.amazonaws.services.lambda.runtime.Context
+import com.amazonaws.services.lambda.runtime.events.SQSEvent
+import graphql.codegen.AddAntivirusMetadata.{AddAntivirusMetadata => av}
```

Comment:
Maybe change import abbreviation to: avm

Reply:
Makes sense, I've changed it.